tests/tests/cpld_control_c48

```diff
@@ -106,6 +106,8 @@ export CHLCLIM=''
 export FRUNOFF=''
 export MOM6_RIVER_RUNOFF=False
 export MOM6_RESTART_SETTING=r
+export MOM6_DIAG_COORD_DEF_Z_FILE=oceanda_zgrid_25L.nc
```

Comment:
May I ask why the C48 test has different diag coord def file from other resolutions?